docs/essentials/toolbars-and-globals.mdx

```diff
@@ -77,22 +79,27 @@ We have a `global` implemented. Let's wire it up! We can consume our new `theme`
   {/* prettier-ignore-end */}
 </IfRenderer>
 
-
 ## Setting globals on a story
 
-By default, custom toolbars and other globals defined with the `globalTypes` annotation, similar to the one created to set up our custom theme, are consistently applied to every story available. However, if you need to, you can override them at the component or story level to use a custom value for a specific story or subset of stories. This will allow you to test specific scenarios (e.g., custom appearance, behavior) where a particular value should be used. To do this, add a `globals` annotation in your story to enable the selected option.
+<Callout variant="warning" icon="🆕">
+  The ability to set globals on a story or component is available in Storybook 8.3+. Some addons, like [backgrounds](./backgrounds.mdx) and [viewport](./viewport.mdx), have been updated to use the `globals` API when a feature flag is enabled.
+</Callout>
+
+When a global value is changed with a toolbar menu in Storybook, that value continues to be used as you navigate between stories. But sometimes a story requires a specific value to render correctly, e.g., when testing against a particular environment.
+
+To ensure that a story always uses a specific global value, regardless of what has been chosen in the toolbar, you can set the `globals` annotation on a story or component. This overrides the global value for those stories and disables the toolbar menu for that global when viewing the stories.
 
 {/* prettier-ignore-start */}
 
-<CodeSnippets path="component-story-theme-override-globals.md" />
+<CodeSnippets path="addon-backgrounds-define-globals.md" />
 
 {/* prettier-ignore-end */}
 
-When Storybook loads the stories, it applies the selected theme to every story except the `Dark` story, which automatically applies the `dark` theme. It also turns off the toolbar dropdown with the theme options, preventing them from being changed and notifying that this story requires a specific value to be used.
+In the example above, Storybook will force all Button stories to use a gray background color, except the `OnDark` story, which will use the dark background. For all Buttons stories, the toolbar menu will be disabled for the `backgrounds` global, with a tooltip explaining that the global is set at the story level.
```

Comment:
When you're able, can you adjust the first paragraph or the one after the example, as we're repeating the same information, specifically that the toolbar is disabled when the user sets the `globals` option with the only caveat that it shows a tooltip.

Reply:
I'm OK with a little repetition here. The first paragraph explains the behavior in general, and the second explains it as it specifically relates to the code snippet. I think having both is helpful.

Reply:
I have no issue with repetition, but my aim here would be to change the wording a bit so that we repeat the exact same information in both places